packages/sdk/src/oracle/services/statemachines/statemachine.ts

```diff
@@ -0,0 +1,106 @@
+import { Context, Memory, ContextType } from "../../types/statemachine";
```

Comment:
this file was moved into its own folder. there are slight modifications to this class to allow multiple types of contexts in the same queue, rather than being instanced previously to a single type. This is so we can have a single queue of contexts regardless of type.